src/main/java/org/truffleruby/extra/ffi/PointerNodes.java

```diff
@@ -168,6 +168,17 @@ public DynamicObject free(DynamicObject pointer) {
 
     }
 
+    @Primitive(name = "pointer_clear", lowerFixnum = 1)
```

Comment:
I prefer treating it as `memset`. If I need to look up the method, "memset" is what I'm going to think of.

Reply:
It no longer has the memset functionality (of setting each byte to a given value), just clearing the bytes to 0.
And this is now part of the PointerNodes, there is no memset-like operation needing for pointers.
We can rename to `bzero` or so but it seems less clear to me.

Reply:
It's not a method, so no need to look this up until you look at code in pointer.rb and there you have the full primitive name.

Reply:
I know it no longer has the same functionality. But I think we should have just kept the previous memset method and implementation. By "look up" I mean when I have the use case of needing to set all the bytes in a buffer, `memset` is what I think of. I think having a general purpose function is more useful here. But, it's not a huge deal.

Reply:
Now it is an internal implementation helper for MemoryPointer's `initialize`, so it is no longer user-visible .
So far, we never needed to set bytes like the previous `memset` on a Pointer instance, so I think it is enough.
But of course we can extend it back if we have the need.
It just seems a strange use-case to set bytes to a value different than 0.